src/config/index.ts

```diff
@@ -6,6 +6,7 @@ import { validateConfig } from './validate.js'
 // Get config from environment variables
 const rawConfig: RawConfig = {
 	port: process.env.PORT || '80',
+	logLevel: process.env.LOG_LEVEL || 'info',
 	appName: process.env.npm_package_name || 'miniurl',
```

Comment:
If you're adding the log-to-file option, we need to add a config flag to enable it (default: disabled).

Reply:
I do not. there is no such option